app/routes/register-your-interest/index.js

```diff
@@ -0,0 +1,81 @@
+const urlPrefix = require('../../config/index').urlPrefix
+const ruralPaymentsAgency = require('../../config/index').ruralPaymentsAgency
+const defraIdConfig = require('../../config').authConfig.defraId
+const BUSINESS_EMAIL_SCHEMA = require('../../schemas/business-email.schema.js')
+const Joi = require('joi')
+
+const ERROR_MESSAGE = {
+  enterYourEmailAddress: 'Enter your business email address',
+  enterYourEmailAddressInCorrectFormat: 'Enter your email address in the correct format, like name@example.com'
+}
+
+module.exports = [
+  {
+    method: 'GET',
+    path: `${urlPrefix}/register-your-interest`,
+    options: {
+      auth: false,
+      handler: async (request, h) => {
+        if (defraIdConfig.enabled) {
+          return h.view('defra-id/register-your-interest/index', { ruralPaymentsAgency })
+        } else {
+          return h.view('register-your-interest/index', { ruralPaymentsAgency })
+        }
+      }
+    }
+  },
+  {
+    method: 'POST',
+    path: `${urlPrefix}/register-your-interest`,
+    options: {
+      auth: false,
+      validate: {
+        payload: Joi.object({
+          emailAddress: BUSINESS_EMAIL_SCHEMA
+            .messages({
+              'any.required': ERROR_MESSAGE.enterYourEmailAddress,
+              'string.base': ERROR_MESSAGE.enterYourEmailAddress,
+              'string.empty': ERROR_MESSAGE.enterYourEmailAddress,
+              'string.email': ERROR_MESSAGE.enterYourEmailAddressInCorrectFormat
+            })
+        }),
+        failAction: async (request, h, error) => {
+          const errorMessages = error
+            .details
+            .reduce((acc, e) => ({ ...acc, [e.context.label]: { text: e.message } }), {})
+          if (defraIdConfig.enabled === true) {
```

Comment:
not sure if we need this check seeing that we will only ever POST to `register-your-interest` once DEFRA ID is enabled. Also we need to make sure when we submit the form we are sending the message to the queue - .e.g.

`await sendRegisterYourInterestMessage(sbi, crn, emailAddress)`